cfgov/jinja2/v1/govdelivery-subscribe/error/index.html

```diff
@@ -20,7 +20,6 @@
         <li>Check back often to follow our progress and find new sources of information.</li>
         <li><a href="/contact-us/">Let us know</a> if there’s a particular line of work you want to hear about regularly.</li>
     </ol>
-    <!-- TODO: Commented out until email signup is ready to use.
```

Comment:
Oh whoops, the top one was temporary because it always failed. I've now removed it entirely. Thanks for catching this.